### PR TITLE
Ready to merge: add new term MONDO:0100175 TTN-related myopathy

### LINKED ISSUE #1731: [NTR/gene]TTN-Related Myopathy
Preferred term label:
TTN-Related Myopathy

Synonyms:
TTN Myopathy
Congenital Myopathy Related to TTN

Definition (free text, please give PubMed ID, if applicable, in format PMID:#######)
TTN-Related Myopathy is a disorder of the musculoskeletal system caused by pathogenic variants in the TTN gene encoding the titin protein expressed in striated muscle. These variants are associated with a variety of overlapping congenital and adult-onset myopathies characterized by non-progressive or progressive neck, axial, and limb weakness, joint contractures, early-onset respiratory insufficiency, facial weakness, congenital cardiac anomalies and/or early-onset dilated cardiomyopathy. Histologic findings on skeletal muscle biopsy reveal a wide range of structural abnormalities and can include increased internalized and central nuclei, minicores, and dystrophic changes (PMID:27854229 and PMID:29691892).

Parent term (use OLS, or your favorite ontology browser):
MONDO:0019952 (congenital myopathy)

Suggested child terms for collapse under new term:
MONDO:0015705 (autosomal recessive centronuclear myopathy)
MONDO:0011362 (hereditary proximal myopathy with early respiratory failure)
MONDO:0012714 (early-onset myopathy with fatal cardiomyopathy)
MONDO:0017939 (classic multiminicore myopathy)

Comment: In the absence of evidence supporting distinct differences in molecular mechanisms between the associated disease entities, as well as considerable phenotypic overlap, these entities can be considered part of a clinical spectrum of TTN-related myopathy.

@nicolevasilevsky: This is a ClinGen EP request (https://clinicalgenome.org/affiliation/40031/).

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1731

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -308367,6 +308369,7 @@ xref: UMLS:C0410204 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:16
 xref: UMLS:C3645536 {source="MONDO:equivalentTo", source="Orphanet:169186"}
 is_a: MONDO:0016191 {source="Orphanet:169186"} ! qualitative or quantitative defects of titin
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:308370]: what is the relationship to this term? should it not be a subclass?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -434215,6 +434219,17 @@ intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc
...
+comment: In the absence of evidence supporting distinct differences in molecular mechanisms between the associated disease entities, as well as considerable phenotypic overlap, these entities can be considered part of a clinical spectrum of TTN-related myopathy.
+synonym: "congenital myopathy related to TTN" EXACT [https://clinicalgenome.org/affiliation/40031/]
+synonym: "TTN myopathy" EXACT [https://clinicalgenome.org/affiliation/40031/]
+is_a: MONDO:0019952 {source="https://clinicalgenome.org/affiliation/40031/"} ! congenital myopathy
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:434229]: these can be N+S conditions

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:308370]: Agreed - I added this as a superclass

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:434229]: Got it - I added the equiv axiom

**Review by nicolevasilevsky — COMMENTED** [review]
I believe this does fall into the disease_series_by_gene pattern - but does that mean we should revise the def? I added the logical def.

**Comment by nicolevasilevsky** [comment]
ready to merge after I fix conflict

**Pull request merged**